frontend/src/pages/ExtractProcess.tsx

```diff
@@ -112,11 +112,13 @@ const ExtractProcess = () => {
       return images;
     };
 
-    convertPdfToImages(pdfFile).then((imgs) => {
+    convertPdfToImages(pdfFile)
+    .then((imgs) => {
       setImages(imgs);
       localStorage.setItem("images", JSON.stringify(imgs));
-    });
-  }, [files]);
+    })
+    .finally(() => handleSubmit())
```

Comment:
Should this be a `finally` or another `then`? 

Do we want to call `handleSubmit` in the event of an error?

Reply:
I think Pri said we are gonna handle errors later. Right now we are focusing on happy path. I’m looking to using a error handling tool like chakra UI’s toast or modal for retries